### DISCUSSION TITLE
Custom validation rules for custom fields

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3629] [Feature Requests]
### Suggest an idea for ExpressionEngine

It would be very helpful to have the ability to validate content against custom rules, defined by regular expressions. A similar feature has Google Forms:

![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/bb2b555a-2ef6-4546-a230-db0a8d336f9c)

**Comment by intoeetive** [comment]
This might be an interesting idea.

Just to make sure, you're speaking about validating the field when the entry is saved/created - and if the value does not match the rules, we don't allow saving the entry and show an error?

@TomJaeger FYI

**Comment by shbchk** [comment]
That is right, Yuri!
